test.js

```diff
@@ -447,3 +451,51 @@ test('alias - require previousId and userId', t => {
     })
   })
 })
+
+test('e2e', async t => {
+  const id = uid(16)
+
+  // Segment Write Key for https://segment.com/segment-libraries/sources/analytics_node_e2e_test/overview
+  const analytics = new Analytics('wZqHyttfRO0KxEHyRTujWZQswgTDZx1N')
+  analytics.track({
+    userId: 'prateek',
+    event: 'E2E Test',
+    properties: {
+      id
+    }
+  })
+  analytics.flush()
+
+  // Give some time for events to be delivered from the API to destinations.
+  await sleep(5000)
+
+  const axiosClient = axios.create({
+    baseURL: 'https://api.runscope.com',
+    timeout: 10 * 1000,
+    headers: { 'Authorization': `Bearer ${process.env.RUNSCOPE_TOKEN}` }
```

Comment:
`'Authorization'` => `authorization` (without quotes)

Reply:
why's that? it should be capitalized?

Reply:
HTTP headers are case insensitive.